src/GeneralStateTestsFiller/stTransactionTest/ValueOverflowFiller.yml

```diff
@@ -0,0 +1,50 @@
+# Send a transaction where value > 256bit
+# i.e. the 257th bit is set, and the value formed by the remaining 256 least significant bits
+# is a number less than or equal to the balance.
+---
+ValueOverflow:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 1000000000
+      code: ''
+      nonce: 0
+      storage: {}
+    d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0:
+      balance: 0
+      code: ''
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      - ''
+    gasLimit:
+      - 21000
+    gasPrice: 100
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: '0xd0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0d0'
+    # Value is (2 ** 256) + 1
+    value:
+      - '0x:bigint 0x10000000000000000000000000000000000000000000000000000000000000001'
```

Comment:
No, I think it should get regected. as all values in Yellow paper are 256 bit max. 
so the balance check should not occur as the structure itself is invalid